### PR TITLE
CVS-60842 task add support for myriad device in docker

### PR DESCRIPTION
**Pull request by jasiu86**
MYRIAD Run:
https://ci.iotg.sclab.intel.com/job/ovmsc/job/OVMSCOps/5340

### CODE REVIEW

**Comment by jasiu86** [comment]
https://ci.iotg.sclab.intel.com/job/ovmsc/job/OVMSCOps/4963/

**Comment by jasiu86** [comment]
MYRIAD Run:
https://ci.iotg.sclab.intel.com/job/ovmsc/job/OVMSCOps/4975/

**Review by rasapala — APPROVED** [review]
LGTM

**Review by jszczepa — APPROVED** [review]
Looks ok. Verification log: http://validationreports.sclab.intel.com:8001/reports/test_results/611f68dbcb116e316c2b2a01

File: tests/functional/object_model/docker.py [hunk 1]
```diff
@@ -85,11 +104,12 @@ def _start(self):
     def stop(self):
         if self.container is not None:
             logger.info(f"Stopping container: {self.container_name}")
+            self.container.stop(timeout=10)
             self.save_container_logs()
-            self.container.remove(force=True)       # Container will be stopped and cleaned
+            self.container.remove(v=True)
```
Comment (ncybulsk) [inline tests/functional/object_model/docker.py:109]: v=? Is it correct?

Comment (jasiu86) [reply tests/functional/object_model/docker.py:109]: Remove Volumes mapping

**Review by ncybulsk — APPROVED** [review]

**Pull request merged**